baseplate/ratelimit.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import division
+
+import time
+
+from .context import ContextFactory
+
+try:
+    from .context.memcache import MonitoredMemcacheConnection
+except ImportError as exc:
+    pass
+
+try:
+    from .context.redis import MonitoredRedisConnection
+except ImportError:
+    pass
```

Comment:
Is there anything more I should be doing for these import errors?

Reply:
You might want to make this a module and put the memcache and redis stuff in it's own files, i.e.:

```
ratelimit/
    __init__.py
    backends/
        __init__.py
        redis.py
        memcache.py
```

Reply:
Then the import errors only happen when you try to use redis or memcache directly, not just from importing the ratelimiter

Reply:
👍